bin/index.js

```diff
@@ -0,0 +1,25 @@
+const dotenv = require("dotenv");
+const envsub = require("envsub");
+const fetch = require("node-fetch");
+dotenv.config();
+
+const [_, __, repoUrlEnv, repoBranchEnv, templatePath, destinationPath] = process.argv;
+(async () => {
+  const commitSha = await fetch(`${process.env[repoUrlEnv]}${process.env[repoBranchEnv]}`)
```

Comment:
can you add the ability to use a GITHUB_TOKEN (ie. the one provided into .env) to make these calls? (if present). you're limited to 60 unauthenticaed calls for 1h and 5k using the token.

Reply:
Fixed in 5a9dcf0